setup/paths.sh

```diff
@@ -37,38 +37,27 @@ else
     source $BUILD_HOME/cmsgemos/setup/paths.sh
 fi
 
+# Export project
+export GEM_PLOTTING_PROJECT=$BUILD_HOME/gem-plotting-tools
```

Comment:
I have the same problem with the name in `vfatqc-python-scripts`, and in `cmsgemos` it has a different purpose